src/utils/permissions.js

```diff
@@ -19,24 +19,20 @@ const hasPermission = (action) => {
   }
 };
 
-// others cannot change the details for devadmin@hgn.net
+
 /**
- * 
+ * Return true if the user is not allowed to update the protected accounts
  * @param {String} devAdminEmail target user email
  * @param {String} authEmail logged in user email
  * @returns {boolean} true if the user is not allowed to update the devadmin details
  */
 export const cantUpdateDevAdminDetails = (devAdminEmail, authEmail) => {
-  const allowedEmails = ['jae@onecommunityglobal.org',
-                         'one.community@me.com',
-                         'jsabol@me.com'
-                        ]
   const protectedEmails = ['jae@onecommunityglobal.org',
                            'one.community@me.com',
                            'jsabol@me.com',
                            'devadmin@hgn.net'
                           ]
-  return protectedEmails.includes(devAdminEmail) && !allowedEmails.includes(authEmail);
+  return protectedEmails.includes(devAdminEmail) && !protectedEmails.includes(authEmail);
```

Comment:
Allowed emails existed because 'devadmin@hgn.net' should be protected, but not able to edit the other protected accounts.